### ISSUE TITLE
Suggested code cleanup: Update all old https://facebook.github.io and https://github.com/facebookincubator/create-react-app links

### ISSUE THREAD

**Issue by sktguha** [facebook/react#19947]
So I suggest replacing the links to https://facebook.github.io and https://github.com/facebookincubator/create-react-app with the correct updated links. 
Yes I understand that redirects are setup properly, so there is no broken links but still it is much cleaner if they had the updated links. For example seeing the old link, a person might go to https://github.com/facebookincubator to find some other important projects there, but of course the important projects like flow type checker etc, are at https://github.com/facebook instead

I think atleast for some places like Readme(these old links are there in some Readme files) it should definitely be done, if not all places
If so, I **want to volunteer** for this effort. Please assign this issue to me in that case

Here is a search for these two links in the codebase: 
https://github.com/facebook/react/search?q=https%3A%2F%2Fgithub.com%2Ffacebookincubator%2Fcreate-react-app
https://github.com/facebook/react/search?q=%22facebook.github.io%22

**Comment by bvaughn** [comment]
Feel free to send a PR, @sktguha

**Comment by sktguha** [comment]
Sure, Thanks @bvaughn . Any order of priority? Like Readme should be done first?
Also can you please assign this issue to me, so anyone else doesn't pick this up by mistake ?
I will start work on this

**Comment by bvaughn** [comment]
No priority.

**Comment by sktguha** [comment]
ok, Got it. Thank you.

**Comment by sktguha** [comment]
@bvaughn I have started work on this
You can see progress on this branch: [https://github.com/sktguha/react/tree/update-outdated-links](url)

Btw should I go ahead and create a PR from this and you can review and merge if ok ? As for the other one, i.e updating facebookincubator/create-react-app links ,  it is bit too long, thus it may turn out to be a big PR if combined at once. 
Maybe I can make another, PR no 2 for those ?

Also I have found some more issues in the codebase and raised another issue regarding them. I think it is important : [https://github.com/facebook/react/issues/19978](url). Please take a look, if you get time.

**Comment by bvaughn** [comment]
One PR or two. It's up to you. When you're ready, just post it and one of us will review it when we get the chance.

**Comment by sktguha** [comment]
Closing this as this has been fixed by the above linked PR
